src/actions.cpp

```diff
@@ -509,7 +518,7 @@ void evm_contract::process_tx(const runtime_config& rc, eosio::name miner, const
     LOGTIME("EVM END");
 }
 
-void evm_contract::pushtx(eosio::name miner, bytes rlptx) {
+void evm_contract::pushtx(eosio::name miner, bytes rlptx, eosio::binary_extension<uint64_t> min_inclusion_price) {
```

Comment:
We must assert that the min_inclusion_price parameter can only be specified when eos_evm_version >= 1.

Reply:
added check to ensure it is either unset or 0, if eos_evm_version is 0. seems like we can't make it unset when generated as inline action in the EOS->EVM bridge transfer notification handler.

Reply:
Lets use `std::optional<uint64_t> min_inclusion_price` in `process_tx`.
Then `dispatch_tx` will call `process_tx` with an empty optional (or an empty binary_extention)